docs/source/application_development/getting_started.rst

```diff
@@ -236,13 +238,62 @@ Alice can grant access to Bob using his public keys:
        label=b'my-secret-stuff',   # Send to Bob via side channel
        m=2,                        # Threshold shares for access
        n=3,                        # Total nodes with shares
+       rate=Web3.toWei(50, 'gwei'),  # 50 Gwei is the minimum rate (per node per period)
        expiration= maya.now() + timedelta(days=5)  # Five days from now
     )
 
    # The policy's public key
    policy_encrypting_key = policy.public_key
 
 
+Putting it all together, here's an example starter script for granting access using a
+software wallet and an existing keyring:
+
+.. code-block:: python
+
+    from nucypher.blockchain.eth.signers import Signer
+    from nucypher.config.keyring import NucypherKeyring
+    from nucypher.characters.lawful import Alice, Bob
+    from umbral.keys import UmbralPublicKey
+    from datetime import timedelta
+    from web3 import Web3
+    import maya
+
+
+    # Restore Existing NuCypher Keyring
+    keyring = NucypherKeyring(account='0xdeadbeef')
```

Comment:
It says above in the document that keyrings are associated with ethereum accounts; will `0xdeadbeef` even work? Maybe we should use the one we created earlier, the `0x287A817...` one?

Reply:
:heavy_check_mark: Good idea - it's more clear to use only one address for alice in these examples.